### ISSUE TITLE
run tests in parallel

### ISSUE THREAD

**Issue by kevgo** [Originate/text-runner#73]
Certain tests like verifying external links can run concurrently.

- only runs X actions in parallel
- displays the parallel actions on the console similar to pnpm

#### parallelizable actions
- check-image
- check-link
- minimum-node-version
- verify-matches-source-code-file
- verify-npm-install
- verify-source-contains-directory

#### approach
- parallelizable actions export 2 functions: `prepare` and `execute`
- or actions export a class that prepares in the constructor and exports a `run` method

#### prerequisites
- [ ]  make all actions non-blocking
- [x] make action naming more consistentc

**Comment by kevgo** [comment]
closing in favor of #96
